src/rgw/rgw_rest_log.cc

```diff
@@ -78,10 +78,15 @@ void RGWOp_MDLog_List::execute() {
   } 
 
   if (period.empty()) {
-    ldout(s->cct, 5) << "Missing period id" << dendl;
-    http_ret = -EINVAL;
-    return;
+    ldout(s->cct, 5) << "Missing period id trying to use current" << dendl;
+    period = store->get_current_period_id();
+    if (period.empty()) {
+      ldout(s->cct, 5) << "Missing period id" << dendl;
+      http_ret = -EINVAL;
+      return;
+    }
```

Comment:
once we reconfigure with a realm and period, our new multisite sync stuff kicks in. can that and the sync agent run at the same time without causing problems?